doc/source/user_guide/fields/field_data.rst

```diff
@@ -59,10 +65,12 @@ by specifying ``FacesNormal`` and ``FacesCentroid`` for ``data_types`` respectiv
 
 .. code-block:: python
 
-  >>> faces_normal_and_centroid_data = field_data.get_surface_data(
+  >>> surface_request_2 = SurfaceFieldDataRequest(
```

Comment:
@prmukherj It looks better if we keep argument ordering consistent between examples.